src/hooks/useActionCenterSubscriber/useLimitOrderActionSubscriber.tsx

```diff
@@ -0,0 +1,394 @@
+  const dispatch = useAppDispatch()
+  const translate = useTranslate()
+  const toastColor = useColorModeValue('white', 'gray.900')
+
+  const {
+    number: { toCrypto },
+  } = useLocaleFormatter()
+
+  const activeQuote = useAppSelector(limitOrderSlice.selectors.selectActiveQuote)
+  const sellAsset = useAppSelector(selectActiveQuoteSellAsset)
+  const buyAsset = useAppSelector(selectActiveQuoteBuyAsset)
+  const sellAmountCryptoBaseUnit = useAppSelector(selectInputSellAmountCryptoBaseUnit)
+  const buyAmountCryptoBaseUnit = useAppSelector(selectBuyAmountCryptoBaseUnit)
+  const quoteId = useAppSelector(selectActiveQuoteId)
+  const { currentData: ordersResponse } = useLimitOrdersQuery()
+  const toast = useToast({
+    render: ({ title, status, description, onClose }) => {
+      const handleClick = () => {
+        onClose()
+        onDrawerOpen()
+      }
+
+      const toastSx = {
+        '&:hover': {
+          cursor: 'pointer',
+          background: status === 'success' ? 'green.300' : 'red.300',
+        },
+      }
+
+      return (
+        <Flex
+          // We can't memo this because onClose prop comes from the render props
+          // eslint-disable-next-line react-memo/require-usememo
+          onClick={handleClick}
+          background={status === 'success' ? 'green.500' : 'red.500'}
+          color='white'
+          px={4}
+          py={2}
+          borderRadius='md'
+          // eslint-disable-next-line react-memo/require-usememo
+          sx={toastSx}
+        >
+          <Box py={1} me={2}>
+            {status === 'success' ? (
+              <Icon color={toastColor} as={TbCircleCheckFilled} height='20px' width='20px' />
+            ) : (
+              <Icon color={toastColor} as={TbCircleXFilled} height='20px' width='20px' />
+            )}
+          </Box>
+          <Box>
+            <CText fontWeight='bold' color={toastColor}>
+              {title}
+            </CText>
+            <CText color={toastColor}>{description}</CText>
+          </Box>
+        </Flex>
+      )
+    },
+  })
+  const openLimitOrders = useAppSelector(selectOpenLimitOrderActionsFilteredByWallet)
+  const quoteExpirationTimestamp = useAppSelector(selectActiveQuoteExpirationTimestamp)
+
+  const orderSubmissionMetadataFilter = useMemo(() => {
+    return { cowSwapQuoteId: quoteId ?? 0 }
+  }, [quoteId])
+
+  const limitOrderSubmissionMetadata = useSelectorWithArgs(
+    selectLimitOrderSubmissionMetadata,
+    orderSubmissionMetadataFilter,
+  )
+
+  const previousLimitOrderState = usePrevious(limitOrderSubmissionMetadata?.state)
+  const previousLimitOrderId = usePrevious(limitOrderSubmissionMetadata?.limitOrder.orderId)
+  const limitPrice = useAppSelector(selectActiveQuoteLimitPrice)
+
+  // Create action after user confirmed the intent of placing a limit order
+  useEffect(() => {
+    if (!limitOrderSubmissionMetadata) return
+    if (!sellAsset) return
+    if (!buyAsset) return
+    if (!quoteId) return
```

Comment:
See comment re: iterating on open IDs - if we do that, this check becomes useless as `quoteId` should always be defined as passed by props

Reply:
I don't know if we want to iterate on open ids for the initial insertion, it would make use adding all the past orders to the action slice

Which could result in instant spam of the action slice with old orders, is that something we want? This sounds like a feature we didn't talk about

quoteId is used to insert the current limit order

Reply:
More like we wouldn't subscribe to something we should skip if that makes sense? i.e similar to `skipToken` logic

Reply:
But subscribing without a quoteId is something we want, for the other actions

The quoteId is only linked to the current limit form, let's discuss this later and open an issue, I'm really not getting this :/